lib/preprocessor.cpp

```diff
@@ -133,22 +133,22 @@ static void inlineSuppressions(const simplecpp::TokenList &tokens, Settings &mSe
         if (!tok->comment)
             continue;
 
-        std::list<Suppressions::Suppression> inlineSuppressions;
-        if (!parseInlineSuppressionCommentToken(tok, inlineSuppressions, bad))
+        std::list<Suppressions::Suppression> inlineSuppr;
```

Comment:
hmm the full name is better imho. Can we change the function name so it says something like `addInlineSuppressions` perhaps? The function name does not tell me what this function does.

Reply:
Renamed the function, changed variable name back.